src/NLog/Config/LoggingRule.cs

```diff
@@ -74,8 +75,8 @@ public LoggingRule(string ruleName)
         /// Create a new <see cref="LoggingRule" /> with a <paramref name="minLevel"/> and  <paramref name="maxLevel"/> which writes to <paramref name="target"/>.
         /// </summary>
         /// <param name="loggerNamePattern">Logger name pattern used for <see cref="LoggerNamePattern"/>. It may include one or more '*' or '?' wildcards at any position.</param>
-        /// <param name="minLevel">Minimum log level needed to trigger this rule.</param>
-        /// <param name="maxLevel">Maximum log level needed to trigger this rule.</param>
+        /// <param name="minLevel">Minimum log level needed to trigger this rule. Log levels range from <see cref="NLog.LogLevel.Trace"/> (lowest) to <see cref="NLog.LogLevel.Fatal"/> (highest).</param>
+        /// <param name="maxLevel">Maximum log level needed to trigger this rule. Log levels range from <see cref="NLog.LogLevel.Trace"/> (lowest) to <see cref="NLog.LogLevel.Fatal"/> (highest).</param>
```

Comment:
Not sure I like to define details LogLevel all around the code. If people need to know of the available LogLevels, then then it should be explained for the LogLevel-type.